hatch/src/assets/config.rs

```diff
@@ -0,0 +1,36 @@
+use project::{ LibraryKind, ProjectKind };
+
+pub struct Config {
+  project: String,
+  lib_type: String,
+}
+
+impl Config {
+  pub fn new(name: &str, project_kind: &ProjectKind) -> Config {
+    Config { project: Config::project(name), lib_type: Config::lib_type(project_kind) }
```

Comment:
Alright I prefer the readability of this but its also documented as the opposite in our style guide, one of them is going to change.

Reply:
It does not explicitly state anything about struct construction :).